### PR TITLE
update examples to the latest API & export AutoSubscribe

### PR DESCRIPTION
**Pull request by theomonnom**

### CODE REVIEW

File: examples/simple-color/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-livekit-agents~=0.7.2
+livekit-agents==0.8.0
```
Comment (keepingitneil) [inline examples/simple-color/requirements.txt:1]: why not ~=

File: examples/simple-color/requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
-livekit-agents~=0.7.2
+livekit-agents~=0.7.2
```
Comment (keepingitneil) [inline examples/simple-color/requirements.txt:1]: why not 0.8.0 ;)

Comment (davidzhao) [inline examples/simple-color/requirements.txt]: should this have changed?

Comment (theomonnom) [reply examples/simple-color/requirements.txt:1]: it is automatically updated

Comment (theomonnom) [reply examples/simple-color/requirements.txt:1]: after the PR gets merged

Comment (theomonnom) [reply examples/simple-color/requirements.txt]: see https://github.com/livekit/agents/blob/df0629912443c9913edf82e5227e2a57d9eb856a/.github/update_versions.py

Comment (theomonnom) [reply examples/simple-color/requirements.txt]: https://github.com/livekit/agents/blob/df0629912443c9913edf82e5227e2a57d9eb856a/.github/update_versions.py

Comment (theomonnom) [reply examples/simple-color/requirements.txt]: It'll get automatically updated after the changesets is merged

**Pull request merged**